Src/IdiomsUnitTest/ConstructorInitializedMemberAssertionTest.cs

```diff
@@ -705,5 +802,11 @@ int IEqualityComparer.GetHashCode(object obj)
                 throw new NotImplementedException();
             }
         }
+
+        enum TestEnum { none = 0, one, two, three };
+
+        enum TestDefaultOnlyEnum { none = 0 };
+
+        enum TestSingleNonDefaultEnum { none = 1 };
```

Comment:
Sorry, am not used to the comment vs line placement.
Thanks, `TestSingleNonDefaultEnum` is indeed missed in my tests.